modules/builds-strategy-docker-build.adoc

```diff
@@ -6,4 +6,4 @@
 [id="builds-strategy-docker-build_{context}"]
 = Docker build
 
-The docker build strategy invokes the docker build command, and it expects a repository with a Dockerfile and all required artifacts in it to produce a runnable image.
+{product-title} uses Buildah to build a container image from a Dockerfile. For more information on building container images with Dockerfiles, see link:https://docs.docker.com/engine/reference/builder/[the Dockerfile reference documentation].
```

Comment:
@adambkaplan I'm worried that this is too generic, and we'll miss on the original motivation here, namely the gotcha wrt having to specify the ARG before the FROM.

I get there there are "a bunch of rules" in the referenced docker documentation, and we don't want to repeat what is there, but this specific ARG/FROM thing has come up enough that IMO it needs some highlighting.

Something like:

`For more information on building container images with Dockerfiles, including rules like the ordering of  'ARG'  commands with respect to 'FROM' commands, see.....`

WDYT?

Reply:
@gabemontero @adambkaplan We can do what Gabe proposes, and/or we can add a new subtopic to the [Troubleshooting builds topic](https://deploy-preview-32982--osdocs.netlify.app/openshift-enterprise/latest/cicd/builds/troubleshooting-builds.html).

Reply:
@gabemontero @adambkaplan  I've added the following tip to cover Gabe's suggestion. WDYT?

Reply:
works for me

LGTM

@adambkaplan ? ... check out https://github.com/openshift/openshift-docs/pull/32982/files#diff-42a3899e5a04b2d96c9f278774c73d775feffbe8a6c652c72da47e8fb5b9326cR11